doc/API.md

```diff
@@ -103,29 +123,62 @@ Content-Length: 156
 Content-Type: application/json; charset=utf-8
 Date: Fri, 23 Sep 2016 16:22:39 GMT
 {
-  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImFkbWluIiwic2NvcGUiOiJhZG1pbiIsImlhdCI6MTQ3NDY0Nzc1OX0.R1vQOLVg8A-6i5QaZQVOGAzImiPvgAdkWiODYhYiNn4"
+  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImFk
+            bWluIiwic2NvcGUiOiJhZG1pbiIsImlhdCI6MTQ3NDY0Nzc1O
+            X0.R1vQOLVg8A-6i5QaZQVOGAzImiPvgAdkWiODYhYiNn4"
 }
 ```
 
 ## GET /auth/facebook
 Authenticates a user using his Facebook account.
 ### Request
-The request must include several query parameters:
-* `authorizationToken`: this contains your client's JWT signed with your secret.
-    Its payload should be `{ id: <key>, scope: 'client' }`.
-* `redirectUrl`: this contains the URL you'd like to be redirected after a
-    successful login. This URL needs to be associated with your client
-    information first.
+Requests must include an JWT signed with a valid client secret as the 
+`authToken` query parameter.
+
+```ssh
+POST /api/auth/facebook?authToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjb
+GllbnRJZCI6IjEyMzQ1Njc4OTAiLCJzY29wZXMiOlsidXNlci1mYXZvcml0ZXMiXSwiYXV0aFJlZ
+GlyZWN0VXJscyI6WyJodHRwczovL2RvbWFpbi5vcmcvYXV0aC9zdWNjZXNzIl0sImF1dGhGYWlsd
+XJlVXJscyI6WyJodHRwczovL2RvbWFpbi5vcmcvYXV0aC9lcnJvciJdfQ.e7rYEZsQNLG0aTjDRH
+sQ2xembu3fyVe-B9bm8mFprwQ HTTP/1.1
+Cache-Control: no-cache
+```
 
-  It will gets the user's JWT as a query parameter `token`.
-* `failureUrl` (optional): this contains the URL you'd like to be redirected
+The payload of the signed JWT must include the following information:
+* `id`: client identifier.
+* `scope`: just `client` for now.
+* `redirectUrl`: the URL you would like to be redirected after a
+    successful login. This URL needs to be associated with your client
+    information first. It will gets the user's JWT as a query parameter `token`.
+* `failureUrl` (optional): the URL you would like to be redirected
     after a failure. This URL needs to be associated with your client
     information first.
 
 ### Response
-A successful request will redirect the user to the facebook login page. Then
-after a successful login the user will eventually be redirected to the URL
-specified in `redirectUrl` with a `token` parameter.
+A successful request will redirect the user to the Facebook login page, where 
+the user should enter her credentials. After a successful login the user 
+will eventually be redirected to the URL specified in `redirectUrl` with a 
+`token` query parameter containing a session token.
+
+```ssh
+https://domain.org/auth/success?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ
+jbGllbnRJZCI6IjEyMzQ1NiIsInNjb3BlIjoidXNlciJ9.nU1LD8gyVxd8kjNJhLlEwhUbohVW3TQ1
+T5hRvYamAiQ
+```
+
+This session token will be in the form of a signed[JWT](https://jwt.io/) with 
+the following data:
+
+```json
+{
+  "id": {
+    "opaqueId": "facebook_id",
+    "provider": "facebook",
+    "clientKey": "02e9c791d7"
+  },
+  "scope": "user"
+}
+```
```

Comment:
nit maybe add a line saying this session token will need to be used for the operations that need a user authentication